CMakeLists.txt

```diff
@@ -300,6 +300,7 @@ add_executable(SystemTestsRunner
     "source/tests/system/niswitch_driver_api_tests.cpp"
     "source/tests/system/niswitch_session_tests.cpp"
     "source/tests/system/nisync_session_tests.cpp"
+    "source/tests/system/nidcpower_driver_api_tests.cpp"
```

Comment:
Same as on the other PR: Keep the driver test files in alphabetical order
```
    "source/tests/system/session_utilities_service_tests.cpp"
    "source/tests/system/nidcpower_driver_api_tests.cpp"
    "source/tests/system/niscope_driver_api_tests.cpp"
    "source/tests/system/niscope_session_tests.cpp"
    "source/tests/system/niswitch_driver_api_tests.cpp"
    "source/tests/system/niswitch_session_tests.cpp"
    "source/tests/system/nisync_session_tests.cpp"
```

Reply:
Done.